_component_design/navigation.md

```diff
@@ -1,9 +1,10 @@
 ---
 title: Navigation
-layout: default
-category: Layout
+parent: null
+layout: component
+category: Components
 usage: Teaser Text
```

Comment:
Have we gotten any text for this metadata from the designers yet?

Reply:
Scratch that; this is a family page. You can remove the `usage` metadata tag; that will only be used on component pages.